### PR TITLE
zh: add enterprise usage

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? (Required)

Add how to deploy enterprise tidb cluster.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-alibaba-cloud.md [hunk 1]
```diff
@@ -191,6 +191,20 @@ category: how-to
 
     根据实际情况修改 `replicas`、`storageClaims[].resources.requests.storage`、`storageClassName`。
 
+    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 spec 下面的 tidb/pd/tikv/tiflash 的 baseImage 项加上 "-enterprise"
```
Comment (DanielZhangQD) [inline zh/deploy-on-alibaba-cloud.md:194]: Suggesting putting this section behind the L230.
We also need to cover the ticdc.

File: zh/deploy-on-aws-eks.md [hunk 2]
```diff
@@ -219,6 +219,20 @@ region = us-west-21
 
     根据实际情况修改 `replicas`、`storageClaims[].resources.requests.storage`、`storageClassName`。
 
+    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 spec 下面的 tidb/pd/tikv/tiflash 的 baseImage 项加上 "-enterprise"
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:222]: ditto

File: zh/deploy-on-gcp-gke.md [hunk 3]
```diff
@@ -180,6 +180,20 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
     参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md)和[集群配置文档](configure-cluster-using-tidbcluster.md)完成 CR 文件配置。
 
+    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 spec 下面的 tidb/pd/tikv/tiflash 的 baseImage 项加上 "-enterprise"
```
Comment (DanielZhangQD) [inline zh/deploy-on-gcp-gke.md:183]:
```suggestion
    值得注意的是，如果需要部署企业版的 tidb/pd/tikv，需要将 db.yaml 中 `spec.<tidb/pd/tikv>.baseImage` 项加上 "-enterprise" 
```

Comment (DanielZhangQD) [reply zh/deploy-on-gcp-gke.md:183]: GKE does not support tiflash/ticdc yet.

File: zh/deploy-on-alibaba-cloud.md (hunk 1 above)
Comment (DanielZhangQD) [reply zh/deploy-on-alibaba-cloud.md:194]:
```suggestion
    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 `spec.<tidb/pd/tikv/tiflash>.baseImage` 项加上 `-enterprise`:
```

File: zh/deploy-on-general-kubernetes.md [hunk 4]
```diff
@@ -69,6 +69,20 @@ TiFlash 支持挂载多个 PV，如果要为 TiFlash 配置多个 PV，可以在
       storageClassName: local-storage
 ```
 
+值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 spec 下面的 tidb/pd/tikv/tiflash 的 baseImage 项加上 "-enterprise"
```
Comment (DanielZhangQD) [inline zh/deploy-on-general-kubernetes.md:72]: Please also update the `upgrade-a-tidb-cluster.md`.

File: zh/deploy-on-alibaba-cloud.md (hunk 1 above)
Comment (ran-huang) [reply zh/deploy-on-alibaba-cloud.md:194]: > Suggesting putting this section behind the L230.
> We also need to cover the ticdc.

Agreed.

File: zh/deploy-tiflash.md [hunk 5]
```diff
@@ -41,6 +41,16 @@ spec:
       storageClassName: local-storage
 ```
 
+值得注意的是，如果需要部署企业版的 tiflash，需要将 db.yaml 中 spec 下面的 tiflash 的 baseImage 项加上 "-enterprise"
```
Comment (ran-huang) [inline zh/deploy-tiflash.md:44]: ditto

File: zh/deploy-on-general-kubernetes.md [hunk 6]
```diff
@@ -69,6 +69,20 @@ TiFlash 支持挂载多个 PV，如果要为 TiFlash 配置多个 PV，可以在
       storageClassName: local-storage
 ```
 
+值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash，需要将 db.yaml 中 spec 下面的 tidb/pd/tikv/tiflash 的 baseImage 项加上 "-enterprise" 
+
+例如:
```
Comment (DanielZhangQD) [inline zh/deploy-on-general-kubernetes.md:74]: Please move this section behind the cdc section.

File: zh/deploy-on-alibaba-cloud.md [hunk 7]
```diff
@@ -214,6 +214,20 @@ category: how-to
 
     根据实际情况修改 `replicas`。
 
+    值得注意的是，如果需要部署企业版的 tidb/pd/tikv/tiflash/ticdc，需要将 db.yaml 中 spec 下面的 `spec.<tidb/pd/tikv/tiflash/ticdc>` 的 baseImage 项加上 "-enterprise"
```
Comment (DanielZhangQD) [inline zh/deploy-on-alibaba-cloud.md:217]:
```suggestion
    值得注意的是，如果需要部署企业版的 TiDB/PD/TiKV/TiFlash/TiCDC，需要将 db.yaml 中 `spec.<tidb/pd/tikv/tiflash/ticdc>.baseImage` 配置为企业版镜像，格式为 `pingcap/<tidb/pd/tikv/tiflash/ticdc>-enterprise`。
```

Comment (DanielZhangQD) [reply zh/deploy-on-alibaba-cloud.md:217]: Discussed with @ran-huang , we think the above suggestion is better, please help update all of the other docs.

File: zh/upgrade-a-tidb-cluster.md [hunk 8]
```diff
@@ -45,6 +45,12 @@ category: how-to
 
     当所有 Pod 都重建完毕进入 `Running` 状态后，升级完成。
 
+> **注意：**
+>
+> 如果需要升级到企业版，需要将 db.yaml 中 `spec.<tidb/pd/tikv/tiflash/ticdc>.baseImage` 配置为企业版镜像，格式为 `pingcap/<tidb/pd/tikv/tiflash/ticdc>-enterprise`。
+>
+> 例如将 `spec.pd.baseImage` 从 `pingcap/pd` 修改为 `pingcap/pd-enterprise`
```
Comment (ran-huang) [inline zh/upgrade-a-tidb-cluster.md:52]:
```suggestion
> 例如将 `spec.pd.baseImage` 从 `pingcap/pd` 修改为 `pingcap/pd-enterprise`。
```

File: zh/upgrade-a-tidb-cluster.md [hunk 9]
```diff
@@ -95,6 +101,12 @@ kubectl annotate --overwrite tc ${cluster_name} -n ${namespace} tidb.pingcap.com
 
     当所有 Pod 都重建完毕进入 `Running` 状态后，升级完成。
 
+> **注意：**
+>
+> 如果需要升级到企业版，需要把 `<tidb/tikv/pd>.image` 的值改为企业版镜像
```
Comment (ran-huang) [inline zh/upgrade-a-tidb-cluster.md:106]:
```suggestion
> 如果需要升级到企业版，需要把 `<tidb/tikv/pd>.image` 的值改为企业版镜像。
```

File: zh/upgrade-a-tidb-cluster.md [hunk 10]
```diff
@@ -95,6 +101,12 @@ kubectl annotate --overwrite tc ${cluster_name} -n ${namespace} tidb.pingcap.com
...
+>
+> 例如将 `pd.image` 从 `pingcap/pd:v4.0.0` 修改为 `pingcap/pd-enterprise:v4.0.0`
```
Comment (ran-huang) [inline zh/upgrade-a-tidb-cluster.md:108]:
```suggestion
> 例如将 `pd.image` 从 `pingcap/pd:v4.0.0` 修改为 `pingcap/pd-enterprise:v4.0.0`。
```

**Comment by DanielZhangQD** [comment]
@lichunzhu Pump is missed in the docs.

**Comment by lichunzhu** [comment]
> @lichunzhu Pump is missed in the docs.

I will fix it.

**Pull request merged**